### ISSUE TITLE
Click on the component will stop animation

### ISSUE THREAD

**Issue by raymanhjn** [chenqingspring/react-lottie#81]
Is that purposed to stop the animation when simply click on that? How should I set up to avoid this situation?

**Comment by igorcoccozup** [comment]
I have the same issue on my animation, whenever I click on it, it will kinda pause/freeze.

**Comment by andrewbdesign** [comment]
I just put `pointer-events: none;` on the element to prevent the pause on click. Curious to know if there is proper way to prevent this without using this css hack.

**Comment by php7webmob** [comment]
Just noticed that you can ommit the {true} in isClickToPauseDisabled={true}. Solved my problem.

**Comment by raymanhjn** [comment]
> It was actually addressed here: #53
> Just add `isClickToPauseDisabled={true}` as a prop fixes this issue.
> 
> For example
> `<Lottie isClickToPauseDisabled={true}/>`

Thank you so much!

**Comment by davebrewer** [comment]
Can you add this to the readme page?
isClickToPauseDisabled={true}

I don't see it there.

**Comment by dowrow** [comment]
Anyone else feels like this is an unexpected default?

**Comment by mosesoak** [comment]
Yes I agree, this is a poor default. Wanting to click an animation to pause it is not an everyday use case, and `isClickToPauseDisabled` is cumbersome at best.

Suggest that Lottie should remove this default and then add a new option like `clickToPause`.

(It's also confusing when some options are props, so the suggestion here would improve that)

**Comment by duplixx** [comment]
please remove this feature as its not good for using in fullscreen window

**Comment by nicholas-ward** [comment]
Agreed. I don't think clicking to pause should be default enabled. This should be by default disabled.
